packages/gas-report/package.json

```diff
@@ -12,14 +12,20 @@
   "exports": {
     ".": "./dist/index.js"
   },
+  "typesVersions": {
+    "*": {
+      "index": [
+        "./dist/index.d.ts"
+      ]
+    }
+  },
   "bin": {
     "gas-report": "./dist/gas-report.js"
   },
   "files": [
     "dist",
     "out",
-    "src",
-    "foundry.toml"
```

Comment:
I think you re-added this in a previous PR right? There are several other packages where this was removed as well, so just wanted to make sure that was intended.

Reply:
yeah I had added one `foundry.toml` in https://github.com/latticexyz/mud/pull/2867 but can't recall if this was motivated by an error that appeared or if I just spotted it was missing and inconsistent with the rest.

but I don't think these are actually needed (they're similar to tsconfig.json)

Reply:
to be certain, I just made a fresh template with this version/PR (published to npm via snapshot action) and confirmed that a local anvil deploy rolled out fine! so I think the resulting app has all it needs inside the `out` dir and doesn't need `foundry.toml` in the npm package

Reply:
Cool! I know I wanted to remove it originally, just wanted to make sure there wasn't some weird reason it needed to be kept.